niceman/retrace/tests/test_packagemanagers.py

```diff
@@ -1,5 +1,6 @@
 # emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
-# ex: set sts=4 ts=4 sw=4 noet:
+# -*- coding: utf-8 -*-
```

Comment:
oh my wonderful mode line for emacs is gone! ;)  I do still use emacs from time to time so why not keep it and just extend with `; coding: utf-8` -- we already have that in few files